src/components/TransactionBlock/Wallet/WalletNftsList.tsx

```diff
@@ -210,6 +210,8 @@ const NftText = styled(Text)<{ marginTop?: number; size?: number }>`
   color: ${({ theme }) => theme.color.text.button};
   ${({ marginTop }) => `margin-top: ${marginTop || 4}px;`}
   ${({ size }) => `font-size: ${size || 12}px;`}
+  ${({ regular }) => !!regular && `font-family: "PTRootUIWebRegular", sans-serif;`}
+  ${({ medium }) => !!medium && `font-family: "PTRootUIWebMedium", sans-serif;`}
```

Comment:
Shall we add the `regular` & `medium` booleans to the attributes of `NftText`?
```
const NftText = styled(Text)<{ marginTop?: number; size?: number, regular?: boolean, medium?: boolean }>
```

Reply:
ah yes good catch